SoftLayer/CLI/globalip/assign.py

```diff
@@ -5,17 +5,17 @@
 
 import SoftLayer
 from SoftLayer.CLI import environment
-from SoftLayer.CLI import helpers
 
 
-@click.command()
+@click.command(epilog="More information about types and ")
 @click.argument('identifier')
-@click.argument('target')
+@click.option('--target',
+              help='See SLDN docs. '
+                   'E.g SoftLayer_Network_Subnet_IpAddress, SoftLayer_Hardware_Server,SoftLayer_Virtual_Guest')
+@click.option('--router', help='An appropriate identifier for the specified $type. Some types have multiple identifier')
```

Comment:
I think calling this option `router` might be confusing, 
```suggestion
@click.option('--target-id', help='The identifier for the destination resource to route this subnet to. ')
```